CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Changes
 
 ### Bugs Fixed
+  * [#816](https://github.com/toptal/chewy/pull/816): Moves query [preference](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/search-search.html#search-preference) from body to query parameters.
```

Comment:
Could you please
```suggestion
  * [#816](https://github.com/toptal/chewy/pull/816): Move query [preference](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/search-search.html#search-preference) from body to query parameters. ([afg419][])
```
and then add
```
[afg419]: https://github.com/afg419
```
at line 628?

Reply:
Yep -- apologies.

Reply:
Thank you very much!